<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>301</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>80</y>
      <width>301</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Setor</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_3">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>30</y>
      <width>291</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Máquina</string>
    </property>
   </widget>
   <widget class="QDateEdit" name="dateEdit">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>80</y>
      <width>110</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_4">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>170</y>
      <width>78</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Corretiva</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_5">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>200</y>
      <width>78</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Preventiva</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>260</y>
      <width>411</width>
      <height>171</height>
     </rect>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Segoe UI'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Descrição do Problema&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>470</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>ENVIAR</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_4">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>130</y>
      <width>301</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Quantidade de Funcionários</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>80</y>
      <width>51</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Baixo</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_2">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>80</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Médio</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_3">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>80</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Alto</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuORDEM_DE_SERVI_O">
    <property name="title">
     <string>Cadastrar OS</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuListar_OS">
    <property name="title">
     <string>Listar OS</string>
    </property>
   </widget>
   <addaction name="menuORDEM_DE_SERVI_O"/>
   <addaction name="menuListar_OS"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
